### PR TITLE
Introduction post for product blog

### PR DESCRIPTION
**Pull request by konklone**
This is my first attempt at an introductory post for our FOIA product blog. 

**[Read the post in full here.](https://github.com/18F/foia/blob/intro/posts/2014-06-23-introduction.md)**

It wouldn't be out of place on 18F's [main blog](http://18fblog.tumblr.com), and probably has some overlap with the post @rjmajma has been working on there. But my goals here are to:
- Introduce the blog, and the repository.
- Communicate the current state of our project plans (as we've communicated internally, and to the task force, and to the rest of 18F) to the rest of the world.

Please use this PR to give feedback on how well it's achieving those, and to make non-controversial or consensus-gained commits on the `intro` branch directly, pre-merge.

In general, I'm hoping we'll all feel free to post as frequently and as technically as we want, without having to schedule time on the 18F blog, or worry about alienating less technical readers. Also, not all posts will be technical -- this one clearly isn't, and instead it serves as a development marker and something we can point to for others.

I'll also add that I've become a big fan of the [open, narrative pull request](https://github.com/18F/foia/pull/4). I'm hoping we'll do a bunch of those, with a positive incentive that some PRs could be quickly adapted into blog posts.

This is on a `gh-pages` branch, because I'm intending this to start out at `https://18f.github.io/foia`. Certainly possible that this changes over time. I don't have an actual Jekyll site, much less a theme, ready, but that will be next and can be set up in parallel to actually approving the content.

Still to do:
- [ ] Create a basic Jekyll site, with theme, and with RSS feed.
- [ ] Add a link to a yet-to-be-created issue/PR around an API specification. (Could be post-publish.)
- [ ] Add a link to our yet-to-be-published 18F blog post about our open source policy. (Could be post-publish.)

### CODE REVIEW

**Comment by konklone** [comment]
I also just added a proposed title, `FOIA and Technology: a development blog`, which is a little bland - suggestions welcome.

File: posts/2014-06-23-introduction.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## FOIA and Technology: a development blog
+
+[18F](https://18f.gsa.gov) has created a delivery team to implement part of the [Open Government National Action Plan](http://www.whitehouse.gov/sites/default/files/docs/us_national_action_plan_6p.pdf)'s commitment to modernizing the Freedom of Information Act.
```
Comment (NoahKunin) [inline posts/2014-06-23-introduction.md]: Overall, fantastic, per usual! I'd simplify and bring some more active conversational voice to the opener. There's something weird on how it reads to me currently - had to pause and re-read it. Even though the rest of the content makes it clear, the sentence right now says the act itself is getting modernized, as opposed to how it works on the ground. I also have direct experience that almost no one knows what "delivery team" means.

What about something like:

Late last year, the Open Government National Action Plan committed to modernizing how the Freedom of Information works. 18F is starting work on the implementation. We wanted to give you an update of where we are and how you can help!

File: posts/2014-06-23-introduction.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+We are tasked with bringing technology to bear on this problem. We are *not tasked* with developing new regulations or legislation. Of course, neither policy nor technology exists in a vacuum, and they will depend heavily on each other.
```
Comment (NoahKunin) [inline posts/2014-06-23-introduction.md]: If someone has "feels" about new regs or legislation mods, should we direct them to that taskforce that was formed recently?

File: posts/2014-06-23-introduction.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+In *this* space ([18f.github.io/foia](https://18f.github.io/foia/)), the delivery team will write about our goals, plans, and progress. Compared to GitHub, updates will be less frequent and more narrative, but this is a **development blog** and so it will frequently be "in the weeds" and technical. You can use [our RSS feed](http://18f.github.io/foia/feed/) to follow our updates.
+
+
+### Our plan
+
+Our team's plan is to **reduce friction** for requestors and agencies in the FOIA process, create **scalable infrastructure** for petitioning a federated government of autonomous agencies, and **clear the path** for the release of public records.
+
+To this end, we're developing, or looking at developing:
+
+* A **consolidated FOIA portal** for requestors. A central, user-facing front-end from which anyone can initiate a FOIA request to an agency in the federal government.
```
Comment (NoahKunin) [inline posts/2014-06-23-introduction.md]: The word "portal" makes ppl think of sites that aren't that great in the gov-context. What about "service" instead? Swapping "service" for "portal" also seems to work in the subsequent bullet points.

Comment (NoahKunin) [inline posts/2014-06-23-introduction.md]: As a separate comment, do we yet have boilerplate text on how this will work [or not] with the two current publicly available "systems"? I think swapping out portal helps alleviate these concerns, but I get questions about what's going to happen to the current sites once a week at this point [and several times at Tcamp!]

File: posts/2014-06-23-introduction.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* A **working API and spec** for agencies to receive FOIA requests and resolve them. In other words, a common way for our portal to cleanly interoperate with an agency's existing or new FOIA processing systems -- at least at the very beginning and end of the process.
+* An interface for **discovering public records** released through FOIA. At the very least, this means a public read API.
+* A lightweight **application for FOIA offices** to track and process requests. It would use the same interoperable request and response API as described above.
+* Tools to help agencies with **efficient preparing of records** for release. In particular, tools to make responsive documents accessible could increase the quality and quantity of publicly released documents &mdash; and be broadly useful in the federal government beyond FOIA.
+
+That's a lot of stuff. We most likely will not end up choosing (or being able) to pursue each of them to completion.
+
+What we think is that these are the areas that have the highest return on investment in using technology to modernize how the Freedom of Information Act functions in the United States.
```
Comment (NoahKunin) [inline posts/2014-06-23-introduction.md]: "What we think is that..." is a bit wordy. Unless we're going to have a long stakeholder evaluation process of changing scope, let's throw a stake in the ground hard define our initial scope. Just start with "These are the areas..." and it will read with greater strength/clarity as well.

File: posts/2014-06-23-introduction.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+And **we welcome your help!** 18F is an [open source team](#) working in the public domain, and if you'd like to be a part of that: here we are.
```
Comment (NoahKunin) [inline posts/2014-06-23-introduction.md]: Potential edit for a stronger call to action:

"...working in the public domain - join us and help government become more transparent and accountable."

File: posts/2014-06-23-introduction.md (hunk 2 above)
Comment (konklone) [inline posts/2014-06-23-introduction.md]: Could you hook me up with a link?

File: posts/2014-06-23-introduction.md (hunk 4 above)
Comment (konklone) [inline posts/2014-06-23-introduction.md]: Totally right on wasted words, will cut. But the above sentence is the truth - this is a wider net than what we'll likely end up doing, with the current size and structure of the project.

File: posts/2014-06-23-introduction.md (hunk 5 above)
Comment (konklone) [inline posts/2014-06-23-introduction.md]: Good call. :+1:

File: posts/2014-06-23-introduction.md (hunk 2 above)
Comment (NoahKunin) [inline posts/2014-06-23-introduction.md]: Sure! This looks to be the best one.

https://ogis.archives.gov/foia-advisory-committee.htm

Comment (rjmajma) [inline posts/2014-06-23-introduction.md]: Do we normally direct people like this (e.g., pointing people who care about procurement in the right direction in FB Open blog posts)? Either way, the Task Force is different than the FACA you link to, but it may be a good place to point folks to. Probably the best option we have at the moment.

Comment (NoahKunin) [inline posts/2014-06-23-introduction.md]: Oh, huh! I didn't know there was both a task force and an advisory group. 

In general, yes, the problem with FBOpen is that there is no useful place to direct people who want to issue recommendations or feedback about procurement. GSA usually sets up a public discussion around specific GWACs, but not about the overall process.

**Comment by konklone** [comment]
Okay, I think I've integrated suggestions from @nsinai, @ErieMeyer, @NoahKunin, and @rjmajma. Collaboration!

I'll plan on merging this later if there aren't further tweaks. Merging doesn't mean it's published, though - I need to create a Jekyll framework for that first. And, pick a header image for the post.

**Comment by NoahKunin** [comment]
So good! Thanks for herding the :cat2:s [i.e. comments]!
